src/Handler/CommandNameExtractor/NamedCommandExtractor.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace League\Tactician\Handler\CommandNameExtractor;
+
+/**
+ * Extract the name from a NamedCommand
+ */
+class NamedCommandExtractor implements CommandNameExtractor
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getNameForCommand($command)
+    {
+        if ($command instanceof NamedCommand) {
+            return $command->getName();
+        }
+
+        // fallback to class name
+        return get_class($command);
```

Comment:
Ok, I won't need this feature. I just thought about it, because of the single-command-bus "priniciple". In that case you have to make ALL your commands a named command.